src/js/components/Aggregate.js

```diff
@@ -90,6 +90,7 @@ export default class Aggregate extends Component {
 Aggregate.propTypes = {
   a11yTitleId: PropTypes.string,
   a11yDescId: PropTypes.string,
+  attribute: PropTypes.string.isRequired,
```

Comment:
There is already a "name" property that should be used for this. Why the addition of "attribute" in addition to "name"?